README.md

```diff
@@ -67,6 +67,13 @@ yarn update-fragment-types
 To release a new version, head over to [Actions](https://github.com/projectcaluma/ember-caluma/actions?query=workflow%3ARelease) and trigger a workflow run.
 This will automatically check if a new version can be released, generate a changelog, draft a release and publish it on NPM.
 
+## Yarn Resolutions
+In order to solve (mostly floating) dependency issues we tend to introduce Yarn resolutions as a pragmatic and temporary solution.
+
+* **focus-trap**  
+  https://github.com/projectcaluma/ember-caluma/pull/1152  
+  Focus trap now throws an error if there is no tabbable element on initialization. This breaks the modal dialogs as they are hidden on initialization.
```

Comment:
Already documented right above the resolution

Reply:
I really hate those 'comments' in JSON :smile:  But I made the change.